README.md

```diff
@@ -80,6 +80,87 @@ Use the command:
 If the test fails, look in the log files in the same directory.
 Here  <install_dir> is the Intel MLSL installation directory.
 
+## Migration to oneCCL ##
+
+Intel® MLSL is no longer supported, no new releases are available, please switch to the new API introduced in Intel® oneAPI Collective Communications Library (oneCCL) http://github.com/intel/oneccl.
+There are some examples that can help you get started with oneCCL, simply try to perform following:
+```
+$ cd ./mlsl_to_ccl
+$ . ${MLSL_ROOT}/intel64/bin/mlslvars.sh
+$ make run -f Makefile_mlsl
+$ . ${CCL_ROOT}/env/vars.sh
+$ make run -f Makefile_ccl
+```
+
+If you used MLSL before, here’s an example of key difference between librares API.
```

Comment:
If you used MLSL before, here is an example that demonstrates the key difference between libraries' APIs.